common/changes/@itwin/core-frontend/eringram-maplayers-formats-promote_2023-08-22-16-21.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Add MapLayerFormat.test.ts with a simple custom implementation of MapLayerFormat using only public/beta APIs, also improve docs in map layer formats-related classes",
```

```suggestion
      "comment": "Promote APIs required to implement a MapLayerFormat to beta",
```
Change log messages are written for end users, who don't care about internal stuff like tests.
It's ok to leave the message empty if only internal changes were made.